types/src/front/lib/assistant.ts

```diff
@@ -84,8 +84,8 @@ export function getLargeWhitelistedModel(
 
 export const GPT_3_5_TURBO_MODEL_ID = "gpt-3.5-turbo" as const;
 export const GPT_4_TURBO_MODEL_ID = "gpt-4-turbo" as const;
-export const GPT_4O_LEGACY_MODEL_ID = "gpt-4o" as const;
-export const GPT_4O_MODEL_ID = "gpt-4o-2024-08-06" as const;
+export const GPT_4O_MODEL_ID = "gpt-4o" as const;
```

Comment:
Shouldn't it be -latest ?

Reply:
no, it's `gpt-4o` ( https://platform.openai.com/docs/models/gpt-4o )

Reply:
:+1: